tests/phpunit/integration/Modules/AnalyticsTest.php

```diff
@@ -417,150 +236,6 @@ function( $caps, $cap ) {
 		}
 	}
 
-	/**
-	 * @dataProvider tracking_disabled_provider
-	 *
-	 * @param array $settings
-	 * @param bool $logged_in
-	 * @param \Closure $assert_opt_out_presence
-	 * @param bool $is_content_creator
-	 */
-	public function test_tracking_disabled( $settings, $logged_in, $is_tracking_active, $is_content_creator = false ) {
```

Comment:
This test is probably worth keeping although it's a bit awkward since opt-out is still printed by `Analytics` but the tags are only printed by `Analytics_4`. Maybe we can re-add it in a future issue when this part is back in the same class?

Reply:
As discussed internally, we have some coverage for the tracking disabled functionality in the `Analytics_4Test` suite. I will include the expansion on this coverage based on the test case being removed on another issue.